keyboardvisibilityevent/src/main/java/net/yslibrary/android/keyboardvisibilityevent/Deregister.java

```diff
@@ -0,0 +1,18 @@
+package net.yslibrary.android.keyboardvisibilityevent;
+
+import android.view.ViewTreeObserver;
+
+/**
+ * @author Anoop S S
+ *         anoopvvs@gmail.com
+ *         on 28/02/2017
+ */
+
+public interface Deregister {
```

Comment:
I think we will go with the common usgea 'unregister'. Will update it with required changes